WaterLog_Backend/Controllers/TankLevelsController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage;
+using Microsoft.Extensions.Configuration;
+using WaterLog_Backend.Models;
+
+namespace WaterLog_Backend.Controllers
+{
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class TankLevelsController : ControllerBase
+    {
+        private readonly DatabaseContext _db;
+        readonly IConfiguration _config;
+        public TankLevelsController(DatabaseContext context, IConfiguration config)
+        {
+            _db = context;
+            _config = config;
+        }
+
+        // GET api/values
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<TankLevelsEntry>>> Get()
+        {
+            return await _db.TankLevels.ToListAsync();
+        }
+
+        // GET api/values/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<TankLevelsEntry>> Get(int id)
+        {
+            return await _db.TankLevels.FindAsync(id);
+        }
+
+        // POST api/values
+        [HttpPost]
+        public async Task Post([FromBody] TankLevelsEntry value)
+        {
+            await _db.TankLevels.AddAsync(value);
```

Comment:
You will probably want to check the ModelState here

Reply:
I try to do it but am not wining please help

Reply:
done